src/Core/ClassInfo.php

```diff
@@ -301,14 +301,14 @@ public static function classImplements($className, $interfaceName)
      */
     public static function classes_for_file($filePath)
     {
-        $absFilePath = Director::getAbsFile($filePath);
+        $absFilePath = Director::getAbsFile(Convert::slashes($filePath));
         $classManifest = ClassLoader::inst()->getManifest();
         $classes = $classManifest->getClasses();
         $classNames = $classManifest->getClassNames();
 
         $matchedClasses = [];
         foreach ($classes as $lowerClass => $compareFilePath) {
-            if (strcasecmp($absFilePath ?? '', $compareFilePath ?? '') === 0) {
+            if (strcasecmp($absFilePath ?? '', Convert::slashes($compareFilePath) ?? '') === 0) {
```

Comment:
I think we need the fallback value inside the `Convert::slashes` call as it expects a string.
```suggestion
            if (strcasecmp($absFilePath ?? '', Convert::slashes($compareFilePath ?? '')) === 0) {
```

Reply:
Done, thanks.